cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -1139,15 +1139,15 @@ def right_scheduler_schema(self, data, **kwargs):
     @validates_schema
     def same_subnet_in_different_queues(self, data, **kwargs):
         """Validate subnet_ids configured in different queues are the same."""
-        for queues in ["slurm_queues", "aws_batch_queues"]:
-            if queues in data:
+        queues = "slurm_queues"
+        if queues in data:
 
-                def _queue_has_subnet_ids(queue):
-                    return queue.networking and queue.networking.subnet_ids
+            def _queue_has_subnet_ids(queue):
+                return queue.networking and queue.networking.subnet_ids
 
-                subnet_ids = {tuple(set(q.networking.subnet_ids)) for q in data[queues] if _queue_has_subnet_ids(q)}
-                if len(subnet_ids) > 1:
-                    raise ValidationError("SubnetIds configured in different queues should be the same.")
+            subnet_ids = {tuple(set(q.networking.subnet_ids)) for q in data[queues] if _queue_has_subnet_ids(q)}
+            if len(subnet_ids) > 1:
+                raise ValidationError("SubnetIds configured in different queues should be the same.")
```

Comment:
I'd write something like: (suggested by Tim) `The SubnetIds used for all of the queues should be the same`

Reply:
Changed